mp4parse/src/lib.rs

```diff
@@ -228,6 +228,7 @@ pub struct AudioSampleEntry {
     pub samplesize: u16,
     pub samplerate: u32,
     pub codec_specific: AudioCodecSpecific,
+    pub protect_info: Option<ProtectionSchemeInfoBox>,
```

Comment:
I don't think this abbreviation is helpful. Please call this field `protection_info`.